src/main/java/net/fabricmc/tinyremapper/TinyRemapper.java

```diff
@@ -956,11 +971,21 @@ private void mrjRefresh(MrjState state) {
 		ClassVisitor visitor = writer;
 
 		if (check) {
-			//noinspection UnusedAssignment
 			visitor = new CheckClassAdapter(visitor);
 		}
 
-		reader.accept(new AsmClassRemapper(visitor, cls.getContext().remapper, rebuildSourceFilenames, checkPackageAccess, skipLocalMapping, renameInvalidLocals), flags);
+		if (post != null) {
+			visitor = post.wrap(visitor, this.getRemapper(), this);
+		}
+
+		visitor = new AsmClassRemapper(visitor, cls.getContext().remapper, rebuildSourceFilenames, checkPackageAccess, skipLocalMapping, renameInvalidLocals);
+
+		if (pre != null) {
+			visitor = pre.wrap(visitor, this.getRemapper(), this);
+		}
```

Comment:
the `this.getRemapper()` does not respect with the mrj version. Shouldn't this be `cls.getContext.remapper`?

Reply:
This PR was made before the MRJ PR was merged so this is probably a merging oversight